src/views/dashboard/ChangeSecret/DataSummary.vue

```diff
@@ -89,5 +89,13 @@ export default {
 
 ::v-deep .content .summary-card {
   flex: 1;
+  width: 33%;
+  max-width: 33%;
+
+  .summary-header {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+  }
 }
 </style>
```

Comment:
Based on the provided code snippet, there does not appear to be any significant difference that would need reviewing or fixing. The code is written in Angular (v2) and follows good practices for CSS classes like `flex` and `width`, making the card responsive while maintaining readability.

However, if you're looking for stylistic improvements or specific design requirements, please specify those details so I can assist more accurately.